src/policy/packages.h

```diff
@@ -88,4 +88,8 @@ bool IsChildWithParents(const Package& package);
  * other (the package is a "tree").
  */
 bool IsChildWithParentsTree(const Package& package);
+
+/** Construct a map from the txid of a transaction to the txids of its in-package ancestor set,
+ * including itself. Input must be IsConsistentPackage, otherwise this returns an empty map. */
```

Comment:
`IsConsistentPackage` in param is Package?
```suggestion
/** Construct a map from the txid of a transaction to the txids of its in-package ancestor set,
 * including itself. Package must be IsConsistentPackage, otherwise this
 * this returns an empty map. Package  */
```

Reply:
This is gone now so marking as resolved